python/setup.py

```diff
@@ -11,5 +11,6 @@
     long_description=open('README.txt').read(),
     url='https://github.com/ROBOTIS-GIT/DynamixelSDK',
     author='Leon Jung',
-    author_email='rwjung@robotis.com'
+    author_email='rwjung@robotis.com',
+    install_requires=['serial']
```

Comment:
@roncapat 

I think it should be changed from serial to pyserial

Reply:
Can be, Pycharm suggested me to install "serial"package but I haven't checked if it was the right name, sorry